android/hybridcommon-ui/src/main/java/com/revenuecat/purchases/hybridcommon/ui/CustomerCenterListenerWrapper.kt

```diff
@@ -0,0 +1,35 @@
+package com.revenuecat.purchases.hybridcommon.ui
+
+import com.revenuecat.purchases.CustomerInfo
+import com.revenuecat.purchases.PurchasesError
+import com.revenuecat.purchases.customercenter.CustomerCenterListener
+import com.revenuecat.purchases.hybridcommon.mappers.map
+
+abstract class CustomerCenterListenerWrapper : CustomerCenterListener {
+
+    override fun onFeedbackSurveyCompleted(feedbackSurveyOptionId: String) {
+        this.onFeedbackSurveyCompletedWrapper(feedbackSurveyOptionId)
+    }
+
+    override fun onRestoreCompleted(customerInfo: CustomerInfo) {
+        this.onRestoreCompletedWrapper(customerInfo = customerInfo.map())
+    }
+
+    override fun onRestoreFailed(error: PurchasesError) {
+        this.onRestoreFailedWrapper(error = error.map().info)
```

Comment:
Hmm any reason we're only sending the userInfo instead of the entire ErrorContainer?

Reply:
I mimicked what we do in paywalls in the equivalent listener wrapper. Maybe we should send the whole error 🤔

Reply:
I reviewed this and when we build the ErrorContainer we add the following as the info

```
mapOf(
            "code" to code.code,
            "message" to message,
            "readableErrorCode" to code.name,
            "readable_error_code" to code.name,
            "underlyingErrorMessage" to (underlyingErrorMessage ?: ""),
        ) + extra
```        

it has nothing to do with `userInfo`. I was also confused by it